docs/pages/web/iconbuttonfloating.js

```diff
@@ -0,0 +1,248 @@
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use IconButtonFloating for positive and supportive actions like Create, Help or Maximize."
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use IconButtonFloating for negative and destructive actions like Delete or Remove. "
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use the designated size and style."
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Scale or style IconButtonFloating. Consistent button sizes promote a cohesive user experience."
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <AccessibilitySection name={generatedDocGen?.displayName}>
+        <MainSection.Subsection
+          title="ARIA attributes"
+          description={`
+IconButtonFloating conveys the component behavior using iconography. IconButtonFloating requires \`accessibilityLabel\`, a text description for screen readers to announce and communicate the represented [Icon](/web/icon). In the example below, the screen reader reads: "Create Pin menu". **The label should describe the intent of the action, not the Icon itself.** For example, use "Edit board" instead of "Pencil".
+
+If IconButtonFloating is used as a control button to show/hide a Popover-based component, we recommend passing the following ARIA attributes to assist screen readers:
+
+- \`accessibilityControls\`: informs the screen reader that IconButtonFloating controls the display of an anchored Popover-based component. It populates [aria-controls](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
+- \`accessibilityHaspopup\`: informs the screen reader that there’s a Popover-based component attached to IconButtonFloating. It populates [aria-haspopup](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
+- \`accessibilityExpanded\`: informs the screen reader whether an anchored Popover-based component is currently open or closed. It populates [aria-expanded](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
+`}
+        >
+          <MainSection.Card cardSize="lg" />
+          <MainSection.Subsection
+            title="Keyboard interaction"
```

Comment:
Is this "Keyboard interaction" or "Landmark" or "Positioning".

Keyboard interaction seems a little bit less explicit. I see the connection but I see some disconnect title<>content